docs/android/emulator/debug-on-emulator.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Debugging on the Android Emulator"
+description: "This guide explains how to launch and debug .NET MAUI apps in Visual Studio using the Android Emulator."
+ms.date: 01/19/2022
+---
+
+# Debug on the Android Emulator
+
+The Android Emulator, installed as part of the **Mobile development with .NET** workload, can be run in various configurations to simulate different Android devices. Each one of these configurations is created as a _virtual device_. In this article, you'll learn how to launch the emulator from Visual Studio and run your app in a virtual device. For more information about how to create and configure a virtual device, see [Android emulator setup](index.md).
+
+## Launching the Emulator
+
+Near the top of Visual Studio, there's the **Solution Configurations** drop-down menu that can be used to select **Debug** or **Release** mode. Choosing **Debug** causes the debugger to attach to the application process running inside the emulator after the app starts. Choosing **Release** mode disables the debugger. When in release mode, you'll need to rely on app logging for debugging.
+
+After you've chosen a virtual device from the **Debug Target** device drop-down menu, select either **Debug** or **Release** mode, then select the Play button to run the application:
+
+:::image type="content" source="media/debug-on-emulator/win/vs-config-selection.png" alt-text="Debug and release modes in Visual Studio along with the Play button.":::
```

Comment:
This is a dark mode screenshot. IDE screenshots are meant to be the default (light) theme, right ("For IDE images, use default colors or themes")?

Reply:
I got this last year too when I did some WPF/WinForms articles. Testing on a fresh VM back then, the default IDE theme was dark.